consensus/proposal/proposal_store_test.go

```diff
@@ -58,17 +52,17 @@ func TestProposalStore_StoreAndGet(t *testing.T) {
 	}{
 		{
 			name:  "store and get single value",
-			key:   createTestHash(1),
+			key:   felt.FromUint64[starknet.Hash](1),
 			value: createTestBuildResult(),
 		},
 		{
 			name:  "store and get multiple values",
-			key:   createTestHash(2),
+			key:   felt.FromUint64[starknet.Hash](2),
 			value: createTestBuildResult(),
 		},
 		{
 			name:  "store with zero hash key",
-			key:   starknet.Hash{},
+			key:   felt.FromUint64[starknet.Hash](0),
```

Comment:
What do you think of adding a default constructor `felt.Default[FeltLike]()` which returns by value?

Reply:
In terms of constructors, I think less is more. I think it's clearer to explicitly see 0 as an argument of the constructor, instead of guessing what the default is (0, 1, mod, etc.)